nakadi-scopes.txt

```diff
@@ -0,0 +1,7 @@
+Resource Type ID: nakadi_event
+Resource Type Name: Nakadi Event
+Owning team: aruha
+Application Scope: nakadi_event.read
```

Comment:
nakadi.event... not nakadi_event.
see https://github.com/zalando/nakadi/pull/86/files#diff-4492c6b26ee0a37518728ec1585420efR112